cmd/debug-cni.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containernetworking/cni/pkg/skel"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
+	cniVersion "github.com/containernetworking/cni/pkg/version"
+	"github.com/containernetworking/plugins/pkg/ns"
+
+	logging "github.com/k8snetworkplumbingwg/cni-log"
+
+	"github.com/maiqueb/fosdem-2023-cni-chaining/pkg/config"
+	"github.com/maiqueb/fosdem-2023-cni-chaining/pkg/version"
+)
+
+func main() {
+	skel.PluginMain(
+		cmdAdd,
+		nil,
```

Comment:
I know that you're just piggybacking kubecon-cni, but it may be worth adding an entry for cmdCheck and just put whatever.

Reply:
I'm OK with it, but, since the purpose of this CNI is to "print" the `prevResult` of chains, the only relevant event is actually `cmdAdd` (the only one with a result).

IMO, I should do the opposite, and undefine `cmdDel`.

Reply:
That's a good point, but I'd keep them basically for the educational part of that. I'm fine if you drop them off but leave a comment.

Reply:
makes sense; I'll add a comment explaining the rationale.